junit-platform-commons/src/main/java/org/junit/platform/commons/util/StringUtils.java

```diff
@@ -70,17 +70,8 @@ public static boolean isNotBlank(String str) {
 	 * @see Character#isWhitespace(int)
 	 */
 	public static boolean containsWhitespace(String str) {
-		if (str == null) {
-			return false;
-		}
+		return str != null && str.codePoints().anyMatch(Character::isWhitespace);
 
```

Comment:
Sorry @marcphilipp, it's not clear to me where you'd like to me insert this blank line. Could you be more specific for me (e.g. before/after a certain line number)?